packages/accordion/src/AccordionDetails.tsx

```diff
@@ -19,18 +19,24 @@ export function AccordionDetails({
 
   /* istanbul ignore next */
   useEffect(() => {
-    // create the observer only once
-    resizeObserver.current = new ResizeObserver(() => {
+    if (typeof ResizeObserver !== 'undefined') {
+      // create the observer only once
+      resizeObserver.current = new ResizeObserver(() => {
+        if (contentRef.current) {
+          setInnerContentHeight(contentRef.current.scrollHeight)
+        }
+      })
+
       if (contentRef.current) {
-        setInnerContentHeight(contentRef.current.scrollHeight)
+        resizeObserver.current.observe(contentRef.current)
+      }
+      return () => {
+        resizeObserver.current?.disconnect()
+      }
+    } else {
+      if (contentRef?.current?.scrollHeight) {
+        setInnerContentHeight(contentRef?.current?.scrollHeight)
```

Comment:
This is the fix, if `ResizeObserver` is `undefined` use `setInnerContentHeight` without `ResizeObserver`

Reply:
Can we add a comment explaining why we'd need to check if ResizeObserver isn't `'undefined'`

Reply:
See [76b2fe9](https://github.com/little-spoon-dev/design-system/pull/817/commits/76b2fe934c5e25afb3ac4369b0ca7f70db30a20b)